Sources/Subprocess/Platforms/Subprocess+Windows.swift

```diff
@@ -155,7 +155,7 @@ extension Configuration {
                                         spawnContext.startupInfo.pointer(to: \.StartupInfo)!,
                                         &processInfo
                                     )
-                                    return (result, processInfo, GetLastError())
+                                    return (result, processInfo, result ? ERROR_SUCCESS : GetLastError())
```

Comment:
Most Windows functions do NOT set GetLastError to 0 on success (and the CreateProcessW docs don't seem to indicate that it is one), so we need to ensure we aren't getting false positive failures, which I saw regularly in release mode.